woodwork/utils.py

```diff
@@ -23,6 +23,41 @@ def import_or_none(library):
         return None
 
 
+def import_or_raise(library, error_msg):
+    '''
+    Attempts to import the requested library.  If the import fails, raises an
+    ImportError with the supplied error message.
+
+    Args:
+        library (str): the name of the library
+        error_msg (str): error message to return if the import fails
+    '''
+    try:
+        return importlib.import_module(library)
+    except ImportError:
+        raise ImportError(error_msg)
+
+
+def import_and_configure_koalas(compute_ops_on_diff_frames=True):
+    """
+    Import the koalas library. Set the compute.ops_on_diff_frames as an optimization.
+
+    Args:
+        compute_ops_on_diff_frames (bool): the koalas compute.ops_on_diff_frames argum,ent
+
+    Returns:
+        A reference to the koalas module
+    """
+    ks = import_or_none('databricks.koalas')
+    if ks is None:
+        return None
+    try:
+        ks.set_option('compute.ops_on_diff_frames', compute_ops_on_diff_frames)
+    except Exception as e:
+        warnings.warn('Unable to set compute.ops_on_diff_frames for koalas: ' + str(e))
+    return ks
```

Comment:
Here's the new logic I added. If the call to `ks.set_option` results in an exception, report it as a warning and continue with the import.

If there's an underlying problem with koalas here, this wouldn't fix that, but it would at least allow the import of woodwork to succeed and log a warning.